resources/views/auth/register/steps/4.blade.php

```diff
@@ -7,7 +7,13 @@
     <div class="field @error('password') field--error @enderror stack">
         <x-hearth-label for="password" :value="__('hearth::auth.label_password')" />
         <x-hearth-hint for="password">
-            {{ __('Passwords must be at least eight characters in length and include at least one uppercase letter, at least one number, and at least one special character.') }}
+            {{ __('For your security, please make sure your password has:') }}
```

Comment:
the `<x-hearth-hint>` component doesn't support multiple block-level elements (in this case, a paragraph and an unordered list). It's probably best to not use the hint component and wrap this all in a `<div>` like this:

```html
<div class="field__hint" id="password-hint">
...
</div>
``